controller/src/main/java/org/jboss/as/controller/logging/ControllerLogger.java

```diff
@@ -3739,4 +3739,12 @@ OperationFailedRuntimeException capabilityAlreadyRegisteredInContext(String capa
 
     @Message(id = 499, value = "There is no satisfactory capability '%s' available to resources with capability scope '%s'. This capability is registered at address(es) '%s', and are not accessible to resources with scope '%s'.")
     IllegalStateException noSatisfactoryCapability(String capability, String scopeName, Set<RegistrationPoint> addresses, String scopeNameAgain);
+
+    @LogMessage(level = WARN)
+    @Message(id = 500, value = "There is no UUID string at '%s'. A new value will be generated.")
+    void uuidIsEmpty(String path, @Cause Throwable throwable);
```

Comment:
@lvydra Would you mind removing the @Cause from the output on both traces?

That will make the warning less scaring for the users, the error doesn't show anything relevant for the users and the trace already contains the file location and the invalid file and or the invalid value.

It should not prevent merging this, but I think it is better if we hide the stack trace here.

Reply:
@yersan updated